OpenEdXMobile/src/main/java/org/edx/mobile/util/JavaUtil.java

```diff
@@ -43,12 +44,19 @@ public static String getMemorySize(long bytes) {
         return String.format("%d MB", mb);
     }
 
+    /**
+     * Convert duration into readable format i-e- hh:mm:ss and return null if duration will be zero
+     * or negative.
```

Comment:
Better description:

`Converts and returns the provided duration into a readable format i.e. hh:mm:ss. Returns null if duration is zero or negative.`